plugins/system/schemaorg/src/Extension/Schemaorg.php

```diff
@@ -537,4 +538,30 @@ protected function isSupported($context)
 
         return false;
     }
+
+    /**
+     * The delete event.
+     *
+     * @param   Ojbject    $event  The event
```

Comment:
```suggestion
     * @param   Object    $event  The event
```
or 
`* @param   Model\AfterDeleteEvent  $event  The event`